CONTRIBUTING.md

```diff
@@ -60,9 +61,76 @@ that PRs will sit open for long periods of time.
+
+3. **Provide detailed information**: In the issue description, clearly state the purpose of the issue include as much information as possible, such as:
+    - Steps to reproduce the issue
+    - Expected behavior
+    - Actual behavior
+    - The version of the operating system and the software you are using
+    - Error messages or logs (if applicable)
+
+4. **Use appropriate labels**: Assign relevant labels to indicate:
+    - (WHY) The purpose or objective of this issue with Objective-level "O" labels like `O: security`, `O: new-feature`, etc.
+    - (WHICH) The part of the system this issue relates to using:
+      - External-level "E" labels if the issue fall outside the current scope of the system and is related to external dependencies or projects like `E: non-cosmos`, `E: no-std` etc.
+      - or "Internal-level "I" labels for anything related to the current scope of the product like `I: logic`, `I: documentation`, etc.
+    - (HOW) If any administrative considerations should be taken into account use Administrative-level "A" labels like `A: help-wanted`, `A: critical`, etc.
+
+This assist us prioritize and categorize your issue more effectively and help others and reviewers understand the type and severity of the issue.
+If the issue you worked on was tagged `A: low-priority`, we'll do our best to
+review it in a timely manner, but please expect longer wait times for a review
+in general. If a low priority issue is important to you, please leave a comment
+explaining why, and we will reprioritize it!
+
+## Pull Requests
+
+If you have write access to the ibc-rs repo, you can directly branch off of `main`.
+This makes it easier for project maintainers to directly make changes to your
+branch should the need arise. Otherwise, check [Forking](#forking) section for instructions
+
+Branch names should be prefixed with the author's name followed by a short description
+of the feature, eg. `name/feature-x`.
+
+Pull requests are made against `main` and are squash-merged into main.
+
+**PRs must:**
+
+- make reference to an issue outlining the context
+- update any relevant documentation and include tests
+- add a corresponding entry in the `.changelog` directory using `unclog`,
+  see the [Changelog](#changelog) section for more details.
+
+Pull requests should aim to be small and self-contained to facilitate quick
+review and merging. Larger change sets should be broken up across multiple PRs.
+Commits should be concise but informative, and moderately clean. Commits will be squashed into a
+single commit for the PR with all the commit messages.
+
+In order to help facilitate the PR review process, tag *one* person as the
+reviewer of the PR. If you are unsure of who to tag, your point of contact on
+the ibc-rs team is always a natural candidate; they'll make sure that the PR gets
+reviewed by whomever is most appropriate to review it.
```

Comment:
Let's remove this; we should tag ourselves since we're the ones who know who's best to handle each PR